models/aeif_cond_alpha.cpp

```diff
@@ -280,6 +280,17 @@ nest::aeif_cond_alpha::Parameters_::set( const DictionaryDatum& d )
 
   if ( Delta_T != 0. && V_peak_ <= V_th )
     throw BadProperty( "V_peak must be larger than threshold." );
+  else if ( Delta_T != 0. )
```

Comment:
This is much better, indeed, I'll change it!
No idea about ``V_peak_``; at first I thought it was because users are not supposed to access the variables directly but since only a few of them are like that, it doesn't make sense... do you want me to remove it? What about ``t_ref_`` 'n co. ?